libs/community/langchain_community/vectorstores/elasticsearch.py

```diff
@@ -413,7 +415,7 @@ class ElasticsearchStore(VectorStore):
         distance_strategy: Optional. Distance strategy to use when
                             searching the index.
                             Defaults to COSINE. Can be one of COSINE,
-                            EUCLIDEAN_DISTANCE, or DOT_PRODUCT.
+                            EUCLIDEAN_DISTANCE, MAX_INNTER_PRODUCT or DOT_PRODUCT.
```

Comment:
nitpick: spelling mistake 
```suggestion
                            EUCLIDEAN_DISTANCE, MAX_INNER_PRODUCT or DOT_PRODUCT.
```